wp-config.php

```diff
@@ -77,7 +77,7 @@
  *
  * @link https://codex.wordpress.org/Debugging_in_WordPress
  */
-define('WP_DEBUG', false);
```

Comment:
Yes! There are lots of ways to set it up i think - the only way i know how is through nginx. [This is how you get the var in php](http://php.net/manual/en/function.getenv.php)